libraries/CurieBLE/examples/Scanning/Scanning.ino

```diff
@@ -0,0 +1,129 @@
+/*
```

Comment:
Can you please put some explanatory comments at the top of each example sketch? i.e. what the sketch does and maybe a quick summary of how it works. Look at the existing built-in example sketches for reference.